homeassistant/components/qbittorrent/strings.json

```
@@ -0,0 +1,27 @@
+{
+    "config": {
+      "step": {
+        "user": {
+          "title": "qBitTorrent",
+          "description": "[%key:common::config_flow::description::confirm_setup%]",
+          "data": {
+            "username": "[%key:common::config_flow::data::username%]",
+            "password": "[%key:common::config_flow::data::password%]",
+            "url": "[%key:common::config_flow::data::url%]"
+          }
+        },
+        "import_confirm": {
+          "title": "Import qbittorret sensor",
+          "description": "A {id} sensor has been discovered in configuration.yaml. This flow will allow you to import it into a config entry."
+        }
```

This should not be needed and thus can be removed

```suggestion
        }
```